composer.json

```
@@ -15,7 +15,7 @@
         }
     },
     "require": {
-        "php": "^7.3",
+        "php": "^7.3 || ^8.0",
```

Please prefer this one:
```suggestion
        "php": ">=7.3",
```

---

See [reasons in this tweet](https://twitter.com/nicolasgrekas/status/1263023258938548225).